src/wwwroot/css/themes/aquamarine.css

```diff
@@ -0,0 +1,66 @@
+:root {
+    /* Core colors */
+    --background: #1a1a1a; /* Dark background */
+    --background-soft: #2a2a2a; /* Slightly lighter dark background */
+    --background-gray: #333333; /* Dark gray */
+    --background-gray-danger: #b22222; /* Dark red for danger */
+    --background-danger: #ff4c4c; /* Bright red for danger */
+    --shadow: rgba(0, 0, 0, 0.7); /* Dark shadow */
+    --light-border: #444444; /* Dark border */
+    
+    /* Text colors */
+    --text: #e0e0e0; /* Light gray for text */
+    --text-soft: #b0b0b0; /* Softer gray for text */
+    --popup-front: #ffffff; /* White for popups */
+    
+    /* UI Elements */
+    --emphasis: #00ffcc; /* Neon teal for emphasis */
+    --emphasis-soft: #66ffe0; /* Softer neon teal */
+    --emphasis-text: #ffffff; /* White for emphasis text */
+    --border-color: #444444; /* Dark border color */
+    
+    /* Buttons */
+    --button-text: #ffffff; /* White for button text */
+    --button-background: #333333; /* Dark button background */

```

Comment:
all these inline comments are redundant... yeah it's a button-background and it's a dark color, it's already there, you don't need to repeat that